src/main.rs

```
@@ -1,3 +1,28 @@
+
 fn main() {
-    println!("Hello, world!");
+    let database = Database::new().expect("Failed to create database!");
+
+    let mut args = std::env::args().skip(1);
+    let key = args.next().expect("TODO: SHOW USAGE");
+    let value = args.collect::<Vec<String>>().join(" ");
+}
+
+struct Database {
+    map: HashMap<String, String>,
```

clippy

field `map` is never read

(warning)